core/src/main/java/org/apache/cxf/common/util/SortedArraySet.java

```diff
@@ -71,25 +71,29 @@ private T[] newArray(int size) {
     }
 
     public boolean add(T o) {
```

Comment:
The changes to `add` aren't fixing any bug. It just seemed like a waste to do a full sort every time.

Reply:
👍